openapi/src/main/java/io/micronaut/openapi/visitor/AbstractOpenApiVisitor.java

```diff
@@ -1005,10 +1005,19 @@ protected void processSchemaProperty(VisitorContext context, TypedElement elemen
                 // check schema required flag
                 AnnotationValue<io.swagger.v3.oas.annotations.media.Schema> schemaAnnotationValue = element.getAnnotation(io.swagger.v3.oas.annotations.media.Schema.class);
                 Optional<Boolean> elementSchemaRequired = Optional.empty();
+                boolean isAutoRequiredMode = true;
                 boolean isRequiredDefaultValueSet = false;
                 if (schemaAnnotationValue != null) {
                     elementSchemaRequired = schemaAnnotationValue.get("required", Argument.of(Boolean.TYPE));
                     isRequiredDefaultValueSet = !schemaAnnotationValue.contains("required");
+                    io.swagger.v3.oas.annotations.media.Schema.RequiredMode requiredMode = schemaAnnotationValue.get("requiredMode", io.swagger.v3.oas.annotations.media.Schema.RequiredMode.class).orElse(null);
```

Comment:
use `enumValue(..)` instead of converting in this way it is less expensive

Reply:
fixed